sympy/stats/tests/test_continuous_rv.py

```diff
@@ -429,7 +429,8 @@ def test_gumbel():
     mu = Symbol("mu")
     x = Symbol("x")
     X = Gumbel("x", beta, mu)
-    assert simplify(density(X)(x)) == exp((beta*exp((mu - x)/beta) + mu - x)/beta)/beta
+    assert simplify(density(X)(x)) == exp(-exp(-(-mu + x)/beta) - (-mu + x)/beta)/beta
```

Comment:
Can we drop the `simplify` here?

Reply:
Agreed, it just slows down the code. This shouldn't have been there. I don't know the reason placing it there.

Reply:
I got it. It shouldn't be dropped. May be, there are comparison issues. Let's keep it there.

Reply:
The tests could be just:
```diff
--- a/sympy/stats/tests/test_continuous_rv.py
+++ b/sympy/stats/tests/test_continuous_rv.py
@@ -443,7 +443,8 @@ def test_gumbel():
     mu = Symbol("mu")
     x = Symbol("x")
     X = Gumbel("x", beta, mu)
-    assert simplify(density(X)(x)) == exp((beta*exp((mu - x)/beta) + mu - x)/beta)/beta
+    assert str(density(X)(x)) == 'exp(-exp(-(-mu + x)/beta) - (-mu + x)/beta)/beta'
+    assert cdf(X)(x) == exp(-exp((mu - x)/beta))
```